docs/user-guide/api.md

```diff
@@ -9,16 +9,16 @@ is clear and human readable.
 1. API should be versioned and self-documented, so that client code generation
 is possible.
 
-> **Version 3** is the current API, all links should be prefixed with `/v3`
+> **Version 4** is the current API, all links should be prefixed with `/v4`
 
 ## AuthN and AuthZ
 
 For Authentication we're using [JSON Web Tokens]. They need to be passed via
 an `Authorization` header. Generating a JWT can be done by visiting our
-`/login` endpoint.
+`/auth/login` endpoint.
```

Comment:
Ehh, we could call it out here.  Hitting login will redirect them back to token.